README.md

```diff
@@ -456,3 +456,21 @@ Just add the domain to your hosts file and give it a try.
 ```
 sudo sh -c "echo '127.0.0.1 phpmyadmin.local' >> /etc/hosts"
 ```
+
+## pgAdmin
+If you need to access the database and you are running postgres, you can use this additional container.
+
+```
+docker-compose up -d pgadmin
+```
+
+Add the domain to your hosts file:
+
+```
+sudo sh -c "echo '127.0.0.1 pgadmin.local' >> /etc/hosts"
+```
+
+
+After you have started the container open in a webbrowser `pgadmin.local`. You have to set a Master password for pgAdmin, maybe 'nextcloud' is good for this environment.
+Now we have to add the database server. Click on `Add New Server` and give it a name, maybe 'nextcloud.local'. Please open the second tab `Connection` and insert 'database-postgres' as the Host name and 'postgres' as the password. Click on `Save`.
```

Comment:
I think it should be possible to provide pgadmin with a custom server list as json file:

PGADMIN_SERVER_JSON_FILE in https://www.pgadmin.org/docs/pgadmin4/latest/container_deployment.html

https://www.pgadmin.org/docs/pgadmin4/latest/import_export_servers.html#json-format

Reply:
According to the docs password fields cannot be imported or exported, but there is a ` "PassFile": "/path/to/pgpassfile",` value that could be useful for this. Not fully sure though

Reply:
Yes :)